pocs/core.py

```diff
@@ -181,6 +182,13 @@ def say(self, msg):
             self.logger.info('Unit says: {}', msg)
         self.send_message(msg, channel='PANCHAT')
 
+        if 'slack_webhook_url' in self.config:
```

Comment:
Some of the slack and twitter stuff got officially pulled into POCS, so let's strip all this from here.

Reply:
OK, removed.